NAMESPACE

```diff
@@ -137,6 +138,7 @@ export(GeomDensity2d)
 export(GeomDotplot)
 export(GeomErrorbar)
 export(GeomErrorbarh)
+export(GeomFreqploy)
```

Comment:
typo?

Reply:
Since I had to create a new class, I had to export this as well.

Reply:
right but shouldn't it be poly? (not ploy)

Reply:
I'm so sorry! Let me correct this asap!